Java/ds/LCAofaBST.java

```diff
@@ -0,0 +1,76 @@
+import java.util.Scanner;
```

Comment:
Add a beginner pitch

Reply:
can you explain this thing , 
what's meant by beginner pitch

Reply:
Beginner friendly description